intro-to-git/collaborating.md

```diff
@@ -0,0 +1,171 @@
+- Explain how to get changes from a remote repository using git pull (or git fetch and git merge)
+- Explain how to push changes to a remote repository using git push
+- Recall that a local machine must merge in changes from the remote before pushing
+
+## Introduction
+
+Developers working on projects tracked by Git build their projects with commits. We can imagine a single developer, Nakita, working on her project named Hike Planner. By the end of the week, she's created twenty commits, and her Git history tells her the story of the code.
+
+How would Nakita's workflow change if the Hike Planner team grows? Multiple developers can work on a project, and Git has even more tools to facilitate collaborating. The key to collaborating with Git is working to converge on a single, unified Git history together.
+
+
+## Vocabulary and Synonyms
+
+| Vocab | Definition | Synonyms | How to Use in a Sentence
+| --- | --- | --- | ---
+Remote repository | asdf | Remote, remote repo, `origin` | asdf
+
+## Collaboration Means Looking at the Same Source of Code
+
+Nakita's friends, Frida and Raul, want to join Nakita and help her build Hike Planner. How does Nakita share the current Git history that's already built? How do Frida and Raul get access to this Git history? Can Frida and Raul work at the same time as Nakita? Do Frida and Raul make commits too? How do all of these commits get received? How do they get shared?
+
+Git is highly configurable, so most of these settings can change. However, in the vast majority of cases, Git enables collaboration with this setup:
+
+- The project should be hosted somewhere, where team members have access. The project will can be called the project "repo."
+- Each developer will download the repo onto their machine using the Git command `clone`. This "download" will carry the Git history
+- All team members should contribute code changes by making commits on their own machines
+- All team members will receive commits and updates from the main hosted repo
+- All team members will send commits and updates to the main hosted repo
+- All team members will follow and repeat this cycle, enabling a smooth team workflow
+
+### Example: The Hike Planner Team
+
+For the Hike Planner team, we might imagine:
+
+- Nakita will host the project codebase somewhere, and give access to Frida and Raul
+- Frida will clone this repo onto her laptop. Raul will clone this repo onto his laptop.
+- Nakita will make code changes and commits on her machine. Frida will make other code changes and commits on her machine. Raul will, too.
+- When Nakita wants the updates and new commits that are on the main hosted repo, she will ask for them. These updates and commits will be applied to the code she currently has on her laptop.
+    - When Frida wants updates, she'll ask for them too. The updates/commits will be applied to her current code.
+    - When Raul wants updates, he'll ask for them, and the updates will be applied to his current code.
```

Comment:
Perhaps condense these lines into "When Frida or Raul want updates, ..."

Reply:
Taking this suggestion until I revisit this section and try to remember why this is commented out...